### ISSUE TITLE
Duplicate skills

### ISSUE THREAD

**Issue by jakobes** [expertanalytics/cvcreator#73]
cvcreator does not check for duplicate technical skills as I learned to my chagrin. 

What is the best solution?
 - Remove duplicates automatically?
 - Throw an error?

**Comment by skavhaug** [comment]
:+1: to automatically removing duplicates.

**Comment by skavhaug** [comment]
Why should the user be alerted? A basic feature of certain aggregates is to ignore duplicates. It's the rules of the aggregation that needs to be refined imo.

**Comment by alexanfl** [comment]
Output a warning, but remove duplicates should be the best solution, no?

**Comment by skavhaug** [comment]
Sounds good to me!

**Comment by jakobes** [comment]
I agree. However there is a risk a warning will drown in the avalanche of output produced by pdflatex.
